### PR TITLE
renew nr updates

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#6552

*Description of changes:*
- see ticket for details
- updated based on new renew designs
- fee summary restructured to make styling easier (currently was not matching designs)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/components/dialogs/reapply.vue
Comment (severinbeauvais) [inline src/components/dialogs/reapply.vue:2]: Does this shrink OK if the user has a narrow viewport?

File: src/components/existing-request/existing-request-display.vue
Comment (severinbeauvais) [inline src/components/existing-request/existing-request-display.vue:319]: Date-mixin methods FTW! 😀

File: src/components/payment/fee-summary.vue
Comment (severinbeauvais) [inline src/components/payment/fee-summary.vue:5]: Why use a v-row here with no v-col in it? Would a div work as well?

File: src/components/payment/fee-summary.vue
Comment (cameron-freshworks) [inline src/components/payment/fee-summary.vue:79]: Just curious, is this additional row required with only one element within?

**Review by cameron-freshworks — APPROVED** [review]
generally looks really good! Just a question about the additional rows within the columns and if they are required beyond giving access to vuetify's helper styling. 
Overall though, all good 👍

File: src/components/dialogs/reapply.vue
Comment (kialj876) [reply src/components/dialogs/reapply.vue:2]: yes it does (I tested it by narrowing my window - do you have a different way of testing this?)

File: src/components/payment/fee-summary.vue
Comment (kialj876) [reply src/components/payment/fee-summary.vue:75]: I thought it was required so I could use the justify "start", but maybe I could do this differently with the v-col. Will check

File: src/components/payment/fee-summary.vue
Comment (kialj876) [reply src/components/payment/fee-summary.vue]: updated - this one, checking to see if I can change the ones cam questioned below

File: src/components/dialogs/reapply.vue
Comment (severinbeauvais) [reply src/components/dialogs/reapply.vue:2]: That's how I test it!

File: src/components/payment/fee-summary.vue
Comment (severinbeauvais) [reply src/components/payment/fee-summary.vue:75]: There's probably a Verify helper class for that

Comment (kialj876) [reply src/components/payment/fee-summary.vue:75]: Alright so I can remove the v-rows for the ones I'm using for the ```justify="start"```, and for the other ones I will add ```style=float: right;``` to the div

Comment (severinbeauvais) [reply src/components/payment/fee-summary.vue:75]: There's probably a float-right class already somewhere.

Comment (kialj876) [reply src/components/payment/fee-summary.vue:75]: I got in the habit of always using v-row for every item on the page because I thought it made adjusting things on the page easier to follow, but I guess it's not really needed and just adds more lines

Comment (severinbeauvais) [reply src/components/payment/fee-summary.vue]: Your other option is to keep the v-rows so your code looks consistent, but add single v-cols under them.

Comment (kialj876) [reply src/components/payment/fee-summary.vue]: haha okay added the float-right class in. I'm good with this unless you would prefer the extra row/cols in there

File: src/components/payment/fee-summary.vue
Comment (severinbeauvais) [reply src/components/payment/fee-summary.vue]: Smells clean now!

(It was a code smell to use v-rows without v-cols, haha.)

File: src/components/payment/fee-summary.vue
Comment (severinbeauvais) [reply src/components/payment/fee-summary.vue]: I'm good with this.

I'll leave it to Cam to resolve this convo.

**Comment by severinbeauvais** [comment]
Ah sorry, one more thing: please update the app version (so we can check it in various environments).

OK, one itty bitty other thing: wait for Cam to deploy his fix from Dev -> Test before you deploy to Dev. (To keep the builds separate, which will allow to deploy to Prod today maybe, without including your change.)

**Review by severinbeauvais — APPROVED** [review]
Approved but see my previous comment before merging.

**Pull request merged**
